notifications_api_common/management/commands/register_kanaal.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+from django.urls import reverse
+
+from ...kanalen import KANAAL_REGISTRY
+from ...models import NotificationsConfig
+
+logger = logging.getLogger(__name__)
+
+
+class KanaalExists(Exception):
+    pass
+
+
+def create_kanaal(api_root: str, kanaal: str) -> None:
+    """
+    Create a kanaal, if it doesn't exist yet.
+    """
+    client = NotificationsConfig.get_client()
+
+    # look up the exchange in the registry
+    _kanaal = next(k for k in KANAAL_REGISTRY if k.label == kanaal)
+
+    kanalen = client.list("kanaal", query_params={"naam": kanaal})
+    if kanalen:
+        raise KanaalExists()
+
+    # build up own documentation URL
+    domain = Site.objects.get_current().domain
+    protocol = "https" if settings.IS_HTTPS else "http"
+    documentation_url = (
+        f"{protocol}://{domain}{reverse('notifications:kanalen')}#{kanaal}"
+    )
+
+    client.create(
+        "kanaal",
+        {
+            "naam": kanaal,
+            "documentatieLink": documentation_url,
+            "filters": list(_kanaal.kenmerken),
+        },
+    )
+
+
+class Command(BaseCommand):
+    help = "Create kanaal in notification component"
+
+    def add_arguments(self, parser):
+        parser.add_argument("kanaal", nargs="?", type=str, help="Name of kanaal")
```

Comment:
Can we make this a "0 or more" type argument? So that we can register all kanalen in one go. Then also rename the command to the plural form.

* `src/manage.py register_kanalen` -> register all channels in `KANAAL_REGISTRY`
* `src/manage.py register_kanalen --kanaal zaken` -> register only "zaken" channel
* `src/manage.py register_kanalen --kanaal zaken --kanaal zaaktypen` -> registers multiple channels.

This *can* be broken up into a separate PR - you can decide.

Reply:
Added it in this PR, as well as the required changes in the OZ pr: https://github.com/open-zaak/open-zaak/pull/1208